make.js

```diff
@@ -45,6 +45,7 @@ exports.makeCombinedAPI = function (apis, sourceDir, apiOutputDir) {
 
     customizations = parseDataFile("customizations.json");
     prereqsAndCleanupMap = parseDataFile("prerequisitesAndCleanup.json");
+    testTokenOverrideMap = parseDataFile("testTokenOverrides.json");

```

Comment:
I wonder if we should combine all this into something like "TestConfig.json"?

Reply:
doesn't have to be this PR btw, just something to think about

Reply:
I did consider that, but I thought it might be slightly easier to navigate the overrides in this file since the prereq/cleanup file will continue to balloon with each test and this one will stay pretty tiny. Then again, once all the tests are set up it'll all stay pretty static anyway. 🤷‍♂️